### DISCUSSION TITLE
Livewire is not defined

### DISCUSSION THREAD

**Discussion by JosephJose** [livewire/livewire#3560] [7. Bugs (archived)]
Hi, if i have to include @livewireScripts at the bottom of the layout page and my livewire components are in the content before livewireScripts, I am getting this error in console: Uncaught ReferenceError: Livewire is not defined

Ok, I added `document.addEventListener("DOMContentLoaded", function(){})` and Livewire loaded, but now I am getting another error "Uncaught TypeError: Livewire.on(...) is not a function"
In console It works...

**Comment by joshhanley** [comment]
@JosephJose can you share some code? Your layout file and an example of your component and script code?

Thanks!

**Comment by SddOffice** [comment]
Uncaught ReferenceError: Livewire is not defined
    at
